application-portal/src/app/services/user/user.service.ts

```diff
@@ -1,9 +1,113 @@
 import { Injectable } from '@angular/core';
+import { User } from '../../models/user/user';
 
 @Injectable({
   providedIn: 'root'
 })
 export class UserService {
+  // initialize variables
+  user: User;
+  id: number;
+  firstName: string;
+  lastName: string;
+  phoneNum: string;
+  email: string;
+  password: string;
+  university: string;
+  gradYear: number;
+  birthDate: string;
 
   constructor() { }
+
+  // Basic Set/Get for user and all fields
+  getUser() {
+    return this.user;
+  }
+
+  setUser(user: User) {
+    this.user = user;
+  }
+
+  getId() {
+    return this.user.id;
+  }
+
+  // not sure when we would use this? but i included it
+  setId(id: number) {
+    this.user.id = id;
+    this.id = id;
+  }
+
+  getFirstName() {
```

Comment:
Are all these User methods necessary? If you're not using them then you don't need to write them :)